vmdb/app/models/mixins/ems_openstack_mixin.rb

```diff
@@ -78,6 +78,14 @@ def verify_api_credentials(options={})
     rescue Excon::Errors::Unauthorized => err
       $log.error("MIQ(#{self.class.name}.verify_api_credentials) Error Class=#{err.class.name}, Message=#{err.message}")
       raise MiqException::MiqEVMLoginError, "Login failed due to a bad username or password."
+    rescue Excon::Errors::Timeout => err
+      $log.error("MIQ(#{self.class.name}.verify_api_credentials) Error Class=#{err.class.name}, Message=#{err.message}")
+      raise MiqException::MiqEVMLoginError, "Login attempt timed out"
```

Comment:
I'm wondering if we just let these errors bubble up, and the UI should use authentication_check instead a la #1990